typings/index.d.ts

```diff
@@ -2184,6 +2187,102 @@ declare module 'discord.js' {
 
   type APIMessageContentResolvable = string | number | boolean | bigint | symbol | readonly StringResolvable[];
 
+  interface APIRawMessage {
+    id: Snowflake;
+    type: number;
+    content: string;
+    channel_id: Snowflake;
+    author: {
+      bot: boolean;
```

Comment:
```suggestion
      bot?: boolean;
```
this is optional, true if present so if you wanted yo be explicit you could do `bot?: true`